src/Documenter.jl

```diff
@@ -325,14 +329,23 @@ function deploydocs(;
 
         repo   = error("no 'repo' keyword provided."),
         branch = "gh-pages",
-        latest = "master",
+        latest::Union{String,Nothing} = nothing, # deprecated
 
         osname = "linux",
         julia  = error("no 'julia' keyword provided."),
 
         deps   = Deps.pip("pygments", "mkdocs"),
         make   = () -> run(`mkdocs build`),
+
+        devbranch = "master",
+        devfolder = "dev",
```

Comment:
I think "directory" is the technically more correct work here, over "folder". So `devdir`? `devdirectory`?  Or even maybe `devurl` or `devpath`, since it's not even a directory here what we're setting, but the URL?

Reply:
`devbranchurl` :stuck_out_tongue: